build/audit.sh

```diff
@@ -31,7 +31,9 @@ SELF=audit
 
 . ./common.sh
 
-if [ -z "${PORTSLIST}" ]; then
+if [ "${PORTSLIST}" ]; then
```

Comment:
that should be:

```suggestion
if [ -n "${PORTSLIST}" ]; then
```

I'm somewhat annoyed at the code growth just to handle external parameter passing and keeping it flexible. Possibly a better solution would be to bounce the configuration files through STAGEDIR?